src/shogun/base/library.cpp

```diff
@@ -0,0 +1,67 @@
+#include <shogun/base/library.h>
+#include <iostream>
+
+namespace shogun
+{
+	class LibraryHandle : public CSGObject
+	{
+	public:
+		LibraryHandle(const std::string& filename)
+		{
+			handle = dlopen(filename.c_str(), RTLD_NOW | RTLD_LOCAL);
+			if (handle) 
+			{
+				std::cout << "Loaded\n";
```

Comment:
Need to remove `iostream`. Do we have logging mechanism or its equivalent in shogun :question: